tags-transformer/README.md

```diff
@@ -0,0 +1,25 @@
+# Rollbar PHP Examples: LaunchDarklyTransformer
+
+This project is an example of how to set up a custom transformer for Rollbar
+reporting for your PHP application. We use this functionality to move the extra
+data sent with the payload to the root level under the `tags` object.
+
```

Comment:
The last sentence ("We use this functionality...") seems to speak in terms of the solution presented, instead of the problem to be solved. I'm expecting the README to tell me "why" the code exists or "what" problem it solves. Is there a more accessible way of expressing the intent?

Reply:
I believe this is done.